util/cron/test-cray-cs-gpu-interop.bash

```diff
@@ -2,6 +2,9 @@
 #
 # GPU interop testing on a Cray CS
 
+# Temporarily use LLVM-13 as a workaround to https://github.com/chapel-lang/chapel/issues/19740
+export USE_LLVM_13=true
```

Comment:
I don't think this is needed for the interop tests because CHPL_LOCALE_MODEL` isn't set to `gpu`.

Reply:
I revised this to remove the changes to common.bash and test-cray-cs-gpu-interop.bash.